src/main/java/org/gridsuite/modification/server/modifications/LineSplitWithVoltageLevel.java

```diff
@@ -28,6 +27,19 @@ public LineSplitWithVoltageLevel(LineSplitWithVoltageLevelInfos modificationInfo
         this.modificationInfos = modificationInfos;
     }
 
+    @Override
+    public void control(Network network) throws NetworkModificationException {
```

Comment:
Add existence check of lineToSplitId ?

Reply:
this is checked at the beginning of apply()